pkg/attestation/crafter/runners/githubaction_test.go

```diff
@@ -118,7 +120,9 @@ func (s *githubActionSuite) TestRunnerName() {
 
 // Run before each test
 func (s *githubActionSuite) SetupTest() {
-	s.runner = NewGithubAction(context.Background())
+	// Create a logger for testing
+	testLogger := zerolog.New(os.Stdout).Level(zerolog.DebugLevel)
```

Comment:
for testing we might want to add the logger that's noOp logger, don't remember the specifics.

Reply:
Yes, something like `log.NewStdLogger(io.Discard)`